backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/OptimizationPipeline.kt

```diff
@@ -42,7 +42,7 @@ private class LlvmPipelineConfiguration(context: Context) {
     val targetTriple: String = context.llvm.targetTriple
 
     // Most of these values are copied from corresponding runtime.bc
-    // Which is using "generic" target CPU for many case.
+    // which is using "generic" target CPU for many case.
     // This approach is suboptimal because target-cpu="generic" limits
     // the set of used cpu features.
     // TODO: refactor KonanTarget so that we can explicitly specify
```

Comment:
Is this comment still correct?

Reply:
"Most" -> "Some" :)
Anyway, this comment is still useful.